ui/lib/apps/SlowQuery/components/SlowQueriesTable.tsx

```diff
@@ -62,7 +62,9 @@ export default function SlowQueriesTable({
       columns={columns}
       items={slowQueries}
       onRowClicked={handleRowClick}
-      getKey={(row) => row?.digest}
+      getKey={(row, idx) =>
+        `${row?.digest}_${row?.timestamp}_${row?.txn_start_ts}_${idx}`
```

Comment:
When there is re-order, the key is changed and it becomes no better than key=idx. I prefer to use digest+timestamp

Reply:
Make sense, will update it soon.